numba/transforms.py

```diff
@@ -110,9 +110,14 @@ def _loop_lift_get_candidate_infos(cfg, blocks, livemap):
 
         [callfrom] = loop.entries   # requirement checked earlier
         an_exit = next(iter(loop.exits))  # anyone of the exit block
-        [(returnto, _)] = cfg.successors(an_exit)  # requirement checked earlier
+        if len(loop.exits) > 1:
+            # Pre-Py3.8 ends up here usually
```

Comment:
"usually"... does that mean sometimes it does not? What happens if so?

Reply:
I need to rephrase this. Py3.8 is never reaching the pre-py3.8 branches. But other side of the branch is possible for python version <3.8